pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -439,7 +440,7 @@ stdenv.mkDerivation (finalAttrs: {
     ++ lib.optional withPam pam
     ++ lib.optional withPCRE2 pcre2
     ++ lib.optional withSelinux libselinux
-    ++ lib.optional withRemote libmicrohttpd
+    ++ lib.optionals withRemote [ libmicrohttpd gnutls ]
```

Comment:
I checked the runtime closure, only systemd-journal-gatewayd and lib/systemd/systemd-journal-remote pull in gnutls, the other stuff still uses openssl.

Reply:
@flokli Do you want me to gate those two elements behind another flag?

Reply:
Not sure I understand - this is behind `withRemote` already - I only checked that it doesn't start linking more systemd components against gnutls and documented my findings (that it doesn't).

Reply:
Discussed privately with flokli, resolving.